be/src/vec/columns/column_vector.cpp

```diff
@@ -327,7 +327,7 @@ ColumnPtr ColumnVector<T>::replicate(const IColumn::Offsets& offsets) const {
 
     // vectorized this code to speed up
     IColumn::Offset counts[size];
-    for (size_t i = 0; i < size; ++i) {
+    for (ssize_t i = 0; i < size; ++i) {
```

Comment:
size is `size_t`, not need change `i` to `sszie_t`

Reply:
We use `offsets[i - 1]` inside this for loop, so it is better to use `ssize_t` for `index -1`.